app/views/announcements/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<!-- Tab navbar -->
+<div class = "sticky-top">
+  <!-- General Nav bar -->
+  <nav class="navbar navbar-expand-lg navbar-light bg-dark p-0">
+    <div class="container-fluid">
+      <a class="navbar-brand" href="#">
+        <%= image_tag "bsfc_logo.png", {:alt => "Berkeley Student Food Collective Logo", :width => 40, :height => 40} %>
+      </a>
+      <ul class="navbar-nav">
+        <li class = "nav-item">
+          <small style = "color: white;">Welcome <%= current_user.name %></small>
+        </li>
+        <!-- <li class="nav-item">
+          <%= link_to "Log Out", logout_path, class: "nav-link text-light" %>
+        </li> -->
+      </ul>
+    </div>
+  </nav>
+
+  <!-- Announcements nav bar -->
+  <div>
+    <!-- Switch between tabs -->
+    <div class="dropdown w-100 px-1 border-bottom border-dark">
+      <a class="btn dropdown-toggle fs-1 fw-bold" href="#" role="button" id="dropdownMenuLink" data-bs-toggle="dropdown" aria-expanded="false">
+        Announcements
+      </a>
+      <ul class="dropdown-menu w-100 " style="box-shadow: 0 20px 20px grey" aria-labelledby="navigation-dropdown">
+        <li><%= link_to "Tasks", tasks_path, class: "dropdown-item fs-1 fw-bold" %></li>
+      </ul>
+    </div>
+
+    <!-- add announcement -->
+    <div class = "d-flex align-items-center justify-content-end px-3 py-2">
+      <button type="button" class="btn btn-sm btn-danger">Post Announcement +</button>
+    </div>
+  </div>
+</div>
+
+<!-- Announcements div -->
+<div id = "announcements-div" class = "container bg-light">
+  <!-- Announcement panel -->
+  <div class = "announcement-panel row px-1 py-1 border-bottom border-top border-dark">
+    
+
+    <!-- Right side of panel -->
+    <div class = "col-11 ps-2 pt-2 pb-3">
+      <div class = "d-flex justify-content-between align-items-center">
+        <h5 class = "fw-bold m-0">This is an announcement!</h5>
+      </div>
+      <p class = "fw-lighter small m-0" style = "line-height: 2;"><small>Posted by Melody 9:00am 12/12/12</small></p>
+      <p class = "m-0" style = "line-height: 1;">
+        <small>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+        </small>
+      </p>
+    </div>
+  </div>
+</div>
+
+<!-- Bootstrap JS -->
```

Comment:
if bootstrap is used, it should be loaded as part of one of the view layouts, NOT in specific views.  this should be moved to layouts/application.html.erb or similar.  if it's already there, there is no need for it here.